docs/source/isaura.rst

```diff
@@ -1,5 +1,168 @@
+
+.. list-table::
+   :widths: 50 100 120
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+
+   * - ``vox_size``
+     - ``[float,float,float]``
+     - X, Y, and Z dimensions of the voxels used in the voxelization of the hits.
+
+   * - ``strict_vox_size``
+     - ``bool``
+     - Flag to indicate if the size of the voxels is forced to be exactly the values provided in the previous argument (*True*),
+
+       or, on the other hand, if they are allowed to change a bit for each track, aiming to optimize the voxelization process (*False*).
+
+   * - ``energy_threshold``
+     - ``float``
+     - If the energy of one of the original end-point voxels is smaller than this value,
+
+       the voxel will be dropped and its energy redistributed to the neighbours.
+
+   * - ``min_voxels``
+     - ``int``
+     - The voxel dropping procedure commented on ``energy_threshold`` can only happen if the number of voxels is larger than the value specified in this argument.
+
+   * - ``blob_radius``
+     - ``float``
+     - Radius of the blobs.
+
+   * - ``max_num_hits``
+     - ``int``
+     - Maximum number of hits for an event to be processed.
+
+       Events with more hits are filtered out and their ID will not appear in the ``Filters/topology_select`` table of the output.
+
+
+.. _Isaura workflow:
+
+Workflow
+--------
+
+The basic idea behind Isaura is to run the "traditional" *Paolina* algorithm over the deconvoluted hits output in :doc:`beersheba`, in order to convert ``events`` into ``tracks`` with ``blobs``.
+
+The first step within the algorithm consists in checking that the number of hits is lower than the value provided in the config file (``max_num_hits``). That argument was introduced because, when running *Paolina* algorithm after :doc:`penthesilea`, there were some events that comprise such large amount of hits that the tracking information extraction took ridiculously long. The following picture shows the number of Penthesilea hits per event (with a different scale) for a typical 24h-long low-background run included in the NEXT-White double-beta analysis [#]_. High energy (trigger2) events usually contain around 200 *Penthesilea hits* (as right panel points out), while there are some of them with more than 10000 hits (illustrated in left panel).
+
+ .. image:: images/isaura/nhits_per_evt_r8571.jpg
+   :width: 1000
+
+The plot also shows that these events only  appear a few times within a 24h-long low-background run (around 0.05% of the total set of events). Rejecting this type of events is not a particularly worrysome issue: they would be thrown away in the posterior analysis, owing to the fact that none of them are exclusively contained inside the fiducial volume. The spatial distributions for one of these events is presented below (where the grey dashed lines illustrate the boundaries of the chamber):
```

Comment:
I would remove the ":" at the end of the paragraph (you don't add them in any of the previous pictures)